web-test-runner.config.mjs

```diff
@@ -9,10 +9,10 @@ export default {
   },
   coverageConfig: {
     threshold: {
-      statements: 80,
+      statements: 70,
       branches: 78,
-      functions: 80,
-      lines: 80
+      functions: 70,
+      lines: 70
     }
```

Comment:
Did we want to commit these changes, or was this simply for testing purposes?

Reply:
This is the change that was suggested by Jason from the meeting that we had. It's to make the test passes with the low test coverage.

Reply:
This can  be reverted after https://github.com/AlaskaAirlines/auro-form/issues/47 gets addressed.